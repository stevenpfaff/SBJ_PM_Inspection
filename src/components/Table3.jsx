import { Table } from 'react-bootstrap'

import tasks from '../data/table3.json'

const Table3 = () => (
  <Table striped bordered hover>
    <thead>
      <tr>
        <th>Task</th>
        <th>OK</th>
        <th>Adjusted</th>
        <th>Replace</th>
        <th>Monitor</th>
        <th>Not Performed</th>
        <th>Notes</th>
      </tr>
    </thead>
    <tbody>
      {tasks.map((task, idx) => (
        <tr key={idx}>
          <td>{task.name}</td>
          <td>
            <input type='checkbox' />
          </td>
          <td>
            <div>
              <input type='checkbox' />
            </div>
          </td>
          <td>
            <input type='checkbox' />
          </td>
          <td>
            <input type='checkbox' />
          </td>
          <td>
            <input type='checkbox' />
          </td>
          <td>

          </td>
        </tr>
      ))}
    </tbody>
  </Table>
)

export default Table3
