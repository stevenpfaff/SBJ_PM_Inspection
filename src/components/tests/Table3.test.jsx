import { render } from '@testing-library/react'

import Table3 from '../Table3'

describe('Table3', () => {
  it('should match snapshot', () => {
    const { container } = render(<Table3 />)

    expect(container).toMatchSnapshot()
  })
})
