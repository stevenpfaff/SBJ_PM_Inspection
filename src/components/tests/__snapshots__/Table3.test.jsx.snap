// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table3 should match snapshot 1`] = `
<div>
  <table
    class="table table-striped table-bordered table-hover"
  >
    <thead>
      <tr>
        <th>
          Task
        </th>
        <th>
          OK
        </th>
        <th>
          Adjusted
        </th>
        <th>
          Replace
        </th>
        <th>
          Monitor
        </th>
        <th>
          Not Performed
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          Tire Pressure / Condition
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
      <tr>
        <td>
          Brakes
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
      <tr>
        <td>
          Drive Shaft
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
      <tr>
        <td>
          Springs
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
      <tr>
        <td>
          Steering Components
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
      <tr>
        <td>
          Coolant
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
      <tr>
        <td>
          Power Steering
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
      <tr>
        <td>
          Check Lights
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
      <tr>
        <td>
          Check Strobe
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
      <tr>
        <td>
          Check Backup Alarm
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
      <tr>
        <td>
          Check Wipers
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
      <tr>
        <td>
          Check DOT
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
      <tr>
        <td>
          Check ANSI
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
      <tr>
        <td>
          Check FRA
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <div>
            <input
              type="checkbox"
            />
          </div>
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
        <td>
          <input
            type="checkbox"
          />
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
